channels/ir.m3u

```diff
@@ -35,19 +45,25 @@ http://d2e40kvaojifd6.cloudfront.net/stream/gem_series/playlist.m3u8
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/ChannelIcons/gemusa-icon.png" group-title="Entertainment",GEM USA
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_usa/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/YWgJdew.png" group-title="",Glory TV
+http://glorytv.ashttp9.visionip.tv/live/glorytv-glorytv-glorytv-hsslive-25f-4x3-SD/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/ft8edTx.png" group-title="Food",HealthMedia
 https://livestream.5centscdn.com/cls092118/8f8400b79584fa2a1315aa5e7612d4f3.sdp/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/clLs5In.jpg" group-title="News",Hispan TV
+https://5a61de8ed719d.streamlock.net:1443/live/smil:live.smil/chunklist.m3u8
```

Comment:
Link is broken (tested in VLC and streamtest.in)

Reply:
Changed to a working stream